inst/examples/safetyGraphics_demo_app.R

```diff
@@ -10,14 +10,109 @@ library(safetyGraphics)
   
   charts$nepexplorerMod$meta <- nepExplorer::meta_nepExplorer
   
-  adam_adlbc <- safetyData::adam_adlbc |>
-    mutate(STRESU = gsub("[\\(\\)]", "", regmatches(PARAM, gregexpr("\\(.*?\\)", PARAM))[[1]]),
-           STRESU = ifelse(PARAM == "Creatinine (umol/L)", "mg/dL", STRESU), # convert creatine to mg/dL
-           AVAL = ifelse(PARAM == "Creatinine (umol/L)", AVAL * .0113, AVAL),
-           PARAM = ifelse(PARAM == "Creatinine (umol/L)", "Creatinine (mg/dL)", PARAM))
+  # Helper functions for unit conversions - shortened names
+  convert_creat_to_mgdl <- function(value) {
+    value / 88.4
+  }
+  
+  convert_bun_to_mgdl <- function(value) {
+    value * 2.86
+  }
+  
+  convert_alb_to_mgdl <- function(value) {
+    value * 100
+  }
+  
+  # Apply unit conversions and data cleaning
+  adam_adlbc_ <- safetyData::adam_adlbc |>
+    mutate(
+      STRESU = gsub("[\\(\\)]", "", regmatches(PARAM, gregexpr("\\(.*?\\)", PARAM))[[1]]),
+      # Use case_when for clearer conditional logic
+      AVAL = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(AVAL),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(AVAL),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(AVAL),
+        TRUE               ~ AVAL
+      ),
+      BASE = case_when(
+        PARAMCD == "CREAT" ~ convert_creat_to_mgdl(BASE),
+        PARAMCD == "BUN"   ~ convert_bun_to_mgdl(BASE),
+        PARAMCD == "ALB"   ~ convert_alb_to_mgdl(BASE),
+        TRUE               ~ BASE
+      ),
+      CHG = case_when(
```

Comment:
Since the same changes are being applied to base and aval, i don't think you'd need a unit conversion on CHG, I think that'd stay the same number since it's same division - double check my thinking though @Lovemore-Gakava, it's the morning and my mind is still getting started

Reply:
@pburnsdata - I appreciate your feedback. The update extends beyond the ratio calculation. It aims to maintain consistency across all variables derived from or associated with BASE or AVAL/LBSTRESN, including CHG, A1LO, A1HI, and others. This ensures that users have reliable data for other various uses outside the BUN/CREAT chart.

Reply:
o I see, the is raw CHG, idk why I thoughtt his was fold change - sounds good!